Create newsletter_30_Dec2021.md and upload images for review (#2232)
alan-turing-institute/the-turing-way: communications/newsletters/newsletter_30_Dec2021.md

Like every year, in this last newsletter of 2021, we want to take time to review the year and express our gratitude to you for being an integral part of *The Turing Way*, especially amidst the pandemic this year. 

Find our ‘thank you’ notes and updates from the community in the newsletter below. 👇

![](images/2021-12-thank-you-1.jpg)

*The Turing Way Community, & Scriberia. (2021). Illustrations from the Turing Way book dashes. Zenodo. https://doi.org/10.5281/zenodo.3332807*

## Community Meetings

### Fireside Chat: Shared Concerns in Open Research Communities

On 10 December 2021, 17:00 - 18:00 UTC, the next fireside chat will take place on the topic "Shared Concerns in Open Research Communities". 
We have allocated 30 minutes after the session (18:00 - 18:30 UTC) to have open discussions with attendees and hear their reflections on the topic.
[Register to join the conversation](https://www.eventbrite.co.uk/e/fireside-chat-shared-concerns-in-open-research-communities-tickets-217634158627?keep_tld=1).

The session will feature Emmy Tsang, Kate Hertweck, Tiago Lubiana, Humberto Julio Debat and Melissa Mendonça.

> They will bring their combined perspectives as community builders, educators, trainers, mentors, researchers, industry professionals and facilitators of open and reproducible practises at both local and global scales.

Reviewer: ```suggestion
They will bring their combined perspectives as community builders, educators, trainers, mentors, researchers, industry professionals and facilitators of open and reproducible practices at both local and global scales.
```

We invite you to host and lead future fireside chats in your network.
[Connect with us](https://github.com/alan-turing-institute/the-turing-way/blob/main/CONTRIBUTING.md#get-in-touch) to discuss more.

![](images/2021-12-firesidechat-2.jpg)

### Collaboration Cafes & Coworking Calls

The last Collaboration Cafe of the year is taking place on **15 December 2021 from 15:00 - 17:00 UTC**.

> Join our project members and core contributors to introduce yourself, about your interests in research and how we can work together.

Reviewer: ```suggestion
Join our project members and core contributors to introduce yourself, your interests in research and how we can work together.
```

If you haven’t yet managed to connect with *The Turing Way* community, this would be a good opportunity.
We will return next year with another season of Collaboration Cafes on **19 January 2022**.

> Find more details about the event and sign up: [hackmd.io/@turingway/collaboration-cafe](https://hackmd.io/@turingway/collaboration-cafe).

Reviewer: ```suggestion
Find more details about the event and sign up here: [hackmd.io/@turingway/collaboration-cafe](https://hackmd.io/@turingway/collaboration-cafe).
```

The last Coworking Call of this year is on 13 December from 11:00 - 12:00 UTC.

> Find details: [hackmd.io/@turingway/coworking-call](https://hackmd.io/@turingway/coworking-call).

Reviewer: ```suggestion
Find details here: [hackmd.io/@turingway/coworking-call](https://hackmd.io/@turingway/coworking-call).
```

> Next year, we will resume hosting them weekly on Mondays starting 10 January 2022.

Reviewer: ```suggestion
Next year, we will resume hosting co-working calls weekly on Mondays starting 10 January 2022.
```

### Take Visible Roles in the Community

![](images/2021-12-pathway-3.jpg)

*The Turing Way Community, & Scriberia. (2021). Illustrations from the Turing Way book dashes. Zenodo. https://doi.org/10.5281/zenodo.3332807*

> As we finish three years of *The Turing Way*, we are reflecting on our growth, as a project, a community and breadth of research interests people represent.

Reviewer: ```suggestion
As we finish three years of *The Turing Way*, we are reflecting on our growth as a project, a community and breadth of research interests people represent.
```

We have started working on our next [governance model](https://github.com/alan-turing-institute/the-turing-way/pull/2036) to ensure that the voices and ideas we represent as a community are presented in the impactful roles and decision-making processes as well.

Many of our community members deliver talks and workshops regularly ([see details](https://zenodo.org/communities/the-turing-way/)), but representation within the community spaces is still limited to a few members.
We want to learn how we can support more people in such visible roles.

> This may include joining Book Dash Committee, hosting Collaboration Cafes or Coworking Calls, leading fireside chats, documenting case studies that reflect their domains, delivering 'just in time' workshops, offering mentorship and more.

Reviewer: ```suggestion
This may include joining the Book Dash Committee, hosting Collaboration Cafes or Coworking Calls, leading fireside chats, documenting case studies that reflect their domains, delivering 'just in time' workshops, offering mentorship and more.
```

> Would you like to take one of these roles or have different ideas to enhance our community member’s visibility? 

Reviewer: ```suggestion
Would you like to take one of these roles or do you have different ideas to enhance our community member’s visibility? 
```

To keep it manageable with the growing number of contributors to the project, we have attributed this release without individual author names, which are separately maintained in the [Record of Contributions in the Afterword](https://the-turing-way.netlify.app/afterword/contributors-record.html) of the book.

### Capture Your Contributions

If your name, personal highlights and specific contributions are not yet recorded, please make a Pull Request on the [contributors.md](https://github.com/alan-turing-institute/the-turing-way/blob/main/contributors.md) document of the repository, or [contact us](https://github.com/alan-turing-institute/the-turing-way/blob/main/CONTRIBUTING.md#get-in-touch) so we can help capture your contributions meaningfully (also see our [Acknowledgement Process](https://the-turing-way.netlify.app/community-handbook/acknowledgement.html)).

![](images/2021-12-recognition-5.jpg)

*The Turing Way Community, & Scriberia. (2021). Illustrations from the Turing Way book dashes. Zenodo. https://doi.org/10.5281/zenodo.3332807*

#### Not a Contributor but Use The Turing Way? Tell us About it!

> If you're not already involved in the community as contributors and you use, share or build upon the project, book or community in any way - We would love to hear from you too!

Reviewer: ```suggestion
If you're not already involved in the community as a contributor but you use, share or build upon the project, book or community in any way - We would love to hear from you too!
```

> For example, many people use the online repository structure in their projects, develop the reproducibility plan using our chapters, reuse the Code of Conduct for their projects, cite *The Turing Way* in their manuscripts or grant proposals, and promote our community practises. 

Reviewer: ```suggestion
For example, many people reuse our online repository structure in their projects, develop a reproducibility plan using our chapters, reuse the Code of Conduct for their projects, cite *The Turing Way* in their manuscripts or grant proposals, and promote our community practices. 
```

You can see responses to [Kirstie's Tweet](https://twitter.com/kirstie_j/status/1326126705270984710?s=20) from the last year and share your response with us under [issue #1659](https://github.com/alan-turing-institute/the-turing-way/issues/1659).

### New Set of Illustrations

As always, our Book Dash participants had an opportunity to work with Scriberia Artists on their chapter and research ideas they were collaborating on.
Katya Balakina and Matt Kemp were available for three days working with us on day-to-day concepts such as documentation, authorship, public engagement, recognition and collaboration, as well as complex topics such as community management, data stewardship, research infrastructure, code publication, peer review and translation workflow.

> We also managed to scribe Turing Way specific illustrations such as the history, vision and Fireside chat event.

Reviewer: ```suggestion
We also managed to scribe Turing Way specific illustrations such as the history, vision and the Fireside chat event.
```

The project was also shortlisted for the OpenUK Award under the ‘Sustainability’ category. 
Additionally, under the ‘young person’ category, Paul Owoicho was shortlisted for his work with *The Turing Way* and Sam van Stroud was awarded in recognition for his work in the Turing Data Stories. 
See details [online](https://openuk.uk/openuk-awards-second-edition-2021/).

Finally, the third nomination of the year for *The Turing Way* is for the Open Publishing Award under the ‘Open Publishing Models’ category.
Winners will be revealed soon, as per [their website](https://openpublishingawards.org/index.html). 

### The Call for Open Life Science’s Next Cohort (OLS-5) has Launched!

[Open Life Science](https://openlifesci.org/) is a 16-week mentoring and training program that upskills individuals in open and reproducible research and empowers them to become open science ambassadors in their communities. 
After successfully concluding four cohorts, 113 projects and 180 project leads, we have opened a call for applications for the next cohort, OLS-5, which will take place from February to June 2022. 

> **Application deadline is 15 January 2021**.

Reviewer: ```suggestion
**Application deadline is 15 January 2022**.
```

- **May**: We hosted the fifth edition of Book Dash virtually, released a new set of illustrations and worked with 15 new participants in drafting, revising and releasing more than 10 chapters with the mentorship of our planning committee members. See [May’s newsletter](https://tinyletter.com/TuringWay/letters/book-dash-report-job-announcements-new-illustrations-and-community-updates).
- **June**: We published 8 chapters (with 20 subchapters) across guides for communication, collaboration and ethical research following the Book Dash in the previous month. Collaboration with the Open Life Science continued to offer mentorship and training to *The Turing Way* community members. See [June’s newsletter](https://tinyletter.com/TuringWay/letters/success-stories-from-book-dash-collaborations-videos-from-past-events-more-community-updates).

> - **July**: Multiple positions and The Alan Turing Institute were announced with allocated time to work on *The Turing Way*. Due to low demand, we stopped hosting onboarding calls and invited feedback on the kinds of community calls we could host to engage with members who would like to stay involved but haven’t found a suitable pathway to engage yet. See [July’s newsletter](https://tinyletter.com/TuringWay/letters/new-job-opportunities-community-call-plans-chapters-and-other-updates).

Reviewer: ```suggestion
- **July**: Multiple positions at The Alan Turing Institute were announced with allocated time to work on *The Turing Way*. Due to low demand, we stopped hosting onboarding calls and invited feedback on the kinds of community calls we could host to engage with members who would like to stay involved but haven’t found a suitable pathway to engage yet. See [July’s newsletter](https://tinyletter.com/TuringWay/letters/new-job-opportunities-community-call-plans-chapters-and-other-updates).
```

- **August**: The JupyterHub team members partnered with *The Turing Way* to support the Community Lead Strategic position funded under the Essential Open Source Software (EOSS) for Science funding from the Chan Zuckerberg Initiative (CZI). See [August’s newsletter](https://tinyletter.com/TuringWay/letters/monthly-round-ups-upcoming-events-jupyterhub-community-strategy-partnership-and-more-updates).

> - **September**: Alongside Kirstie, Malvika was promoted to co-lead *The Turing Way* as the Tools, Practices and Systems Senior Researcher, and a position was announced for the recruitment of a new community manager. See [September’s newsletter](https://tinyletter.com/TuringWay/letters/we-are-hiring-meet-us-at-the-fireside-chat-community-share-out-and-contributor-celebration-date).

Reviewer: ```suggestion
- **September**: Alongside Kirstie, Malvika was promoted to co-lead for *The Turing Way* as the Tools, Practices and Systems Senior Researcher, and a position was announced for the recruitment of a new community manager. See [September’s newsletter](https://tinyletter.com/TuringWay/letters/we-are-hiring-meet-us-at-the-fireside-chat-community-share-out-and-contributor-celebration-date).
```
